models/mnist_gan_graph.py

```diff
@@ -0,0 +1,232 @@
+import numpy as np
+import tensorflow as tf
+import tensorflow.contrib.summary as tf_summary
+
+from models.base import BaseModel
+
+# Flags
+from flags import flags_parser
+FLAGS = flags_parser.FLAGS
+assert FLAGS is not None
+
+
+# Utils
+def next_batch(arr, batch_size):
+    num_batches = int(len(arr) / batch_size)
+    for i in range(0, num_batches * batch_size, batch_size):
+        yield arr[i:i + batch_size]
+    yield arr[num_batches * batch_size:]
+
+
+def noise(size, dist='uniform'):
+    if dist == 'uniform':
+        return np.random.uniform(-1, 1, size=size)
+    elif dist == 'normal':
+        return np.random.normal(size=size)
+    elif dist == 'linspace':
```

Comment:
TODO(jendelel): Find a better to do this. Perhaps interpolate the "diagonal". This certainly isn't ideal.

Reply:
Done.